lib/js/jsbigints.nim

```diff
@@ -35,19 +35,23 @@ func toString*(this: JsBigInt; radix: int): cstring {.importjs: "#.$1(#)".} =
   runnableExamples:
     doAssert big"2147483647".toString(2) == "1111111111111111111111111111111".cstring
 
-func toString*(this: JsBigInt): cstring {.importjs: "#.toString()".}
+func toString*(this: JsBigInt): cstring {.importjs: "#.toString()".} # asserted on $
```

Comment:
I don't understand what you mean by `asserted on $` ?

Reply:
`$toString(this)` uses `toString`  :)